### ISSUE TITLE
[Android] PrintJobOrientation _TypeError (type 'Null' is not a subtype of type 'int')

### ISSUE THREAD

**Issue by AliYar-Khan** [pichillilorenzo/flutter_inappwebview#2413]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

It crashes on printing the web page using the example here
https://inappwebview.dev/docs/webview/print-job-controller/

### Expected Behavior

It should print the web page to pdf

### Steps with code example to reproduce

<details>
<summary>Steps with code example to reproduce</summary>

```dart
IconButton(
              onPressed: () async {
                // dispose any previous print job
                printJobController?.dispose();

                final jobSettings = PrintJobSettings(
                    handledByClient: true,
                    jobName:
                        "${await webViewController?.getTitle() ?? ''} - PDF Document example",
                    colorMode: PrintJobColorMode.MONOCHROME,
                    outputType: PrintJobOutputType.GRAYSCALE,
                    orientation: PrintJobOrientation.LANDSCAPE,
                    numberOfPages: 1);

                printJobController = await webViewController?.printCurrentPage(
                    settings: jobSettings);

                if (defaultTargetPlatform == TargetPlatform.iOS) {
                  printJobController?.onComplete = (completed, error) async {
                    if (completed) {
                      if (kDebugMode) {
                        print("Print Job Completed");
                      }
                    } else {
                      if (kDebugMode) {
                        print("Print Job Failed $error");
                      }
                    }
                    printJobController?.dispose();
                  };
                }

                final jobInfo = await printJobController?.getInfo();
                if (kDebugMode) {
                  print(jobInfo);
                }
              },
              icon: const Icon(Icons.print))
        ],
```
</details>

### Stacktrace/Logs

<details>
<summary>Stacktrace/Logs</summary>

```
<Replace this line by pasting your stacktrace or logs here>
```
</details>

### Flutter version

3.24.4

### Operating System, Device-specific and/or Tool

Flutter 3.24.4 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 603104015d (3 weeks ago) • 2024-10-24 08:01:25 -0700
Engine • revision db49896cf2
Tools • Dart 3.5.4 • DevTools 2.37.3

### Plugin version

v6.1.5

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by AliYar-Khan** [comment]
@pichillilorenzo how to use this fix ? use the beta version ?

**Comment by pichillilorenzo** [comment]
Currently on master branch only, new beta version will be released as soon as possible

**Comment by AliYar-Khan** [comment]
@pichillilorenzo still getting issue on beta version

**Comment by pichillilorenzo** [comment]
Beta version 6.2.0-beta.2 is not yet published. You can try to use the master branch.

**Comment by pichillilorenzo** [comment]
Released new version `6.2.0-beta.2` with the fix.
